larndsim/detsim_ep.py

```diff
@@ -50,12 +51,11 @@ def time_intervals(self, event_id_map, tracks, fields):
         t_start = ep.maximum(ep.full_like(tracks_t_start, self.time_interval[0]),
                              ((tracks_t_start - self.time_padding) / self.t_sampling) * self.t_sampling)
         t_length = t_end - t_start
-        track_starts = (t_start + event_id_map_ep * self.time_interval[1] * 3).raw
+        track_starts = t_start.raw
```

Comment:
Where is this change coming from?

Reply:
If we are calculating loss per event, track_starts won't need to be shifted according to their event id. In another word, `event_id_map_ep` will always be 0. 
In the extreme scenario, when there are tracks from multiple events leaked into the calculation. You'd rather them collapse in the same time window, then have a long scrolling time tick list which would mess up the current setup of the translation from time to drift distance z.

Reply:
Ok, I think this makes sense, and I see you've removed `event_id_map` from the arguments, so that's consistent